session_security/static/session_security/script.js

```diff
@@ -15,6 +15,9 @@ if (window.yourlabs == undefined) window.yourlabs = {};
 //   onbeforeunload handler that doesn't block expire().
 // - events: a list of event types to watch for activity updates.
 // - returnToUrl: a url to redirect users to expired sessions to. If this is not defined we just reload the page
+// - noReload: If this is defined then we expire the session without reloading
+//      the page.  Useful if the page should stay visible but no further actions
+//      should be taken.
```

Comment:
I don't understand the last sentence, what do you mean stay visible ?

Isn't there a way to at least encrypt the html until the user logs in again for example rather than taking no further action ?